tests/conftest.py

```diff
@@ -1,4 +1,5 @@
-"""Ruff is forcing me to write a docstring for conftest.py."""
+"""Fixtures used in our test suite for the pyOpenSci Python package 
+template."""
```

Comment:
Docstrings style-wise are preferred to have a single summary line. Maybe:

```suggestion
"""Provide fixtures to the pyOpenSci package template test suite."""
```